dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Protections/DistanceProtectionLineSide1LoadBlinder.mo

```diff
@@ -0,0 +1,157 @@
+within Dynawo.Electrical.Controls.Protections;
+
+/*
+* Copyright (c) 2022, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+
+model DistanceProtectionLineSide1LoadBlinder "Simple triangular distance relay, disconnects side 1 of the protected line when the apparent impedance in within a protected zone for longer than a set duration"
```

Comment:
Maybe we could have a base model that is shared with DistanceProtectionLineSide1 ?

Reply:
It looks like the only difference is Blinded. We could have a unique model and force Blinded to false in the first case ?